src/cpp/polars/Series.cpp

```diff
@@ -505,6 +505,33 @@ namespace polars {
         return m;
     }
 
+    bool Series::empty() const {
+        return (index().is_empty() & values().is_empty());
+    }
+
+    Series Series::head(int rows) const  {
+        Series ser = Series(values(), index());
```

Comment:
This is very pythonic... `Series ser(values(), index());` is more c++

Reply:
Done. Can't get used ot it :(